src/main/java/com/harrison/cb/CouchbaseClient.java

```diff
@@ -19,6 +19,7 @@
 public class CouchbaseClient {
 	
 	
+    @Deprecated
 	public List<String> doCbStuff() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-54-245-133-150.us-west-2.compute.amazonaws.com:9000/sonar/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-54-245-133-150.us-west-2.compute.amazonaws.com:9000/sonar/coding_rules#rule_key=squid%3AS1133)